### PR TITLE
Refactor axis [12022]

### PR DESCRIPTION
**Pull request by jparisu**
Reduce space of time in X axis (horizontal)
Normalize Y axis
Improve chart space efficiency
reresize when max data in window has passed
Y axis from 0
Set Y min and max
Button to interact with the chartview
Button to reset zoom

Signed-off-by: jparisu <javierparis@eprosima.com>

### CODE REVIEW

File: qml/StatisticsChartView.qml [hunk 1]
```diff
@@ -23,33 +23,36 @@ ChartView {
...
+    Component.onCompleted: {
+        axisY.applyNiceNumbers()
+    }
+
     ValueAxis {
         id: axisY
-        min: axisYMin
-        max: axisYMax
+        min: 0
+        max: 1
```
Comment (rsanchez15) [inline qml/StatisticsChartView.qml:45]: Fix the number of labels to 5.

```suggestion
        max: 1
        tickCount: 5
```

**Review by rsanchez15 — CHANGES_REQUESTED** [review]
Some bugs encountered:
- Dynamic charts: if the Y axis is set manually, it comes back to the default values when the chart continues plotting new points.
- Historic charts: The zoom box, drag and zoom in and zoom out does not work if axis values are set manually using the new dialog.

Also remove all qml traces (console.log)

File: qml/StatisticsChartView.qml (hunk 1 above)
Comment (rsanchez15) [reply qml/StatisticsChartView.qml:45]: Set also:
```
minorTickCount: 5
onTickCountChanged: tickCount = 5
```

Comment (jparisu) [reply qml/StatisticsChartView.qml:45]: with `fineNumbers` is automaticly fit to round numbers, we may want to leave it this way. (I change it anyway).

Comment (jparisu) [reply qml/StatisticsChartView.qml:45]: This looks like it does not work with fineNumbers

**Comment by rsanchez15** [comment]
For the rebase: 
- Keep the "incoming change"
- That will cause an error which can be solved applying the following patch:
  ```
  diff --git a/qml/StatisticsChartView.qml b/qml/StatisticsChartView.qml
  index b2e73ce..7b31965 100644
  --- a/qml/StatisticsChartView.qml
  +++ b/qml/StatisticsChartView.qml
  @@ -143,7 +143,7 @@ ChartView {
               }
 
               onExited: {
  -                chartViewMouseAreaItem.cursorShape = Qt.ArrowCursor
  +                chartViewMouseArea.cursorShape = Qt.ArrowCursor
               }
 
               onPressed: {
  ```

**Review by rsanchez15 — APPROVED** [review]
LGTM

**Pull request merged**